tests/core/contracts/test_contract_buildTransaction.py

```diff
@@ -1,19 +1,43 @@
-# -*- coding: utf-8 -*-
-
 import pytest
 
 from eth_utils.toolz import (
     dissoc,
 )
 
+from web3 import Web3
 from web3.exceptions import (
     ValidationError,
 )
+from web3.middleware import (
+    local_filter_middleware,
+)
+from web3.providers.eth_tester import (
+    EthereumTesterProvider,
+)
+
+# -*- coding: utf-8 -*-
+
 
 # Ignore warning in pyethereum 1.6 - will go away with the upgrade
```

Comment:
Looks like this is a duplicate, and we don't need it anyway, so let's delete lines 21-26